mitmproxy/contentviews/img/metadata.py

```diff
@@ -0,0 +1,30 @@
+from . import png
+
+class Metadata:
+    def __init__():
+        pass
+
+    def get_png_metadata():
```

Comment:
Multiple functions please so that they can be easily tested! :smiley:
```python
Metadata = typing.List[typing.Tuple[str, str]]

def parse_png(data: bytes) -> Metadata:
    # extract metadata here
```